Jenkinsfile

```diff
@@ -193,7 +193,7 @@ stage('Publish') {
                     // Use `npm` rather than `yarn` for publishing.
                     // See https://github.com/yarnpkg/yarn/pull/3391.
                     sh "echo '//registry.npmjs.org/:_authToken=${env.NPM_TOKEN}' >> \$HOME/.npmrc"
-                    sh "npm publish --tag ${npmTag}"
+                    sh "yarn publish --tag ${npmTag}"
```

Comment:
There is a comment just above this change stating that the use of `npm publish` rather than `yarn publish` is intentional. In the discussion for a PR there is a link to this commit that explains the reason: https://github.com/hypothesis/client/commit/975843f3ee0d6924479f2f5fa59a66a563db63ea

Does this issue still apply? If so, we should avoid this change. If it no longer applies, we should update the comment above. Also, what about the `.npmrc` line above? Is that the right place to write a token that `yarn publish` uses.

Reply:
Based on this https://github.com/yarnpkg/yarn/pull/5108, I modified the command.

I tested it and I confirm it works as expected. 

I removed the outdated comment and I simplified the way NPM_TOKEN is passed to the yarn command.